scheduler/scheduler_sync.daemon

```diff
@@ -52,6 +51,7 @@ readonly RADAR_ID=$(echo "${RADAR_NAME}" | cut --fields 1 --delimiter '_')
 
 # Import some required settings for sending files over the internet
 source "${HOME}/.scheduler"
+source "${HOME}/.profile"
```

Comment:
It would be a good idea to add the required fields from `.profile` to the dependencies list in the header

Reply:
I believe they already are